### PR TITLE
Add Katello 4.14.1 changelog and doc entry

### PR DESCRIPTION
**Pull request by chris1984**
#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by chris1984** [comment]
@Lennonka can this get a review please?

File: guides/doc-Release_Notes/topics/katello-4.14.1.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+== Katello
+
+* pass:[Container push sometimes makes duplicate repos due to race condition] - https://projects.theforeman.org/issues/37785[#37785]
+* pass:[Deleting published repos from product page doesn't work right] - https://projects.theforeman.org/issues/37782[#37782]
+* pass:[Handle empty CVE in InfoProvider content_view_info] - https://projects.theforeman.org/issues/37779[#37779]
+* pass:[Use :default_location_subscribed_hosts in registration] - https://projects.theforeman.org/issues/37703[#37703]
+* pass:[Pagination within Packages wizard is wonky] - https://projects.theforeman.org/issues/37587[#37587]
+
+=== Content Credentials
```
Comment (Lennonka) [inline guides/doc-Release_Notes/topics/katello-4.14.1.adoc:13]:
```suggestion
=== Content credentials
```
Unless the headings are generated, we use sentence case. Please also correct for other headings.

**Review by Lennonka — COMMENTED** [review]
Generally looks good, just a small nonblocking suggestion.

File: guides/doc-Release_Notes/topics/katello-4.14.1.adoc (hunk 1 above)
Comment (chris1984) [reply guides/doc-Release_Notes/topics/katello-4.14.1.adoc]: @Lennonka they are generated but fixed them anyways, will raise an issue against the redmine script. Let me know how this looks :)

**Pull request merged**